runtime/near-evm-runner/src/pricer.rs

```diff
@@ -0,0 +1,427 @@
+// Below is part of Open Ethereum:
+// Copyright 2015-2020 Parity Technologies (UK) Ltd.
+
+// Open Ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Open Ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Open Ethereum.  If not, see <http://www.gnu.org/licenses/>.
+
+use eth_pairings::public_interface::eip2537::{
+    SCALAR_BYTE_LENGTH, SERIALIZED_G1_POINT_BYTE_LENGTH, SERIALIZED_G2_POINT_BYTE_LENGTH,
+};
```

Comment:
Let's just copy the constants here instead of creating a dependency.
We can attribute to the source

Reply:
Sounds good